src/psyclone/gocean1p0.py

```diff
@@ -2227,14 +2182,14 @@ def lower_to_language_level(self):
 
             # Use a CodeBlock to encode a Fortran pointer assignment
             reader = FortranStringReader(
```

Comment:
Since we're here, this should (if possible) use the PSyIR Fortran frontend rather than explicitly using the various fparser classes, e.g.:

    reader = FortranReader()
    
    cblock = reader.psyir_from_statement(
            f"{vsym_ptr.name} => function_space_collection%get_fs( mesh, "
            f"{order.name}, {space})", table)

Reply:
Yes, that seems to work, at least the tests are passing :)

Reply:
In that case it's right ;-)